client/src/components/HomeScreen.tsx

```diff
@@ -285,6 +298,26 @@ const CreateAudit: React.FC = () => {
                 selectedValue={values.auditType}
               >
                 <Radio value="BALLOT_POLLING">Ballot Polling</Radio>
+                {values.auditType === 'BALLOT_POLLING' ? (
+                  <BallotPollingWrapper>
+                    <label htmlFor="ballotPollingType">
+                      <p>Ballot polling type</p>
+                      <RadioGroup
+                        name="ballotPollingType"
+                        onChange={e =>
+                          setFieldValue(
+                            'ballotPollingType',
+                            e.currentTarget.value
+                          )
+                        }
+                        selectedValue={values.ballotPollingType}
+                      >
+                        <Radio value="BRAVO">BRAVO</Radio>
+                        <Radio value="MINERVA">Minerva (Not recommended)</Radio>
+                      </RadioGroup>
+                    </label>
+                  </BallotPollingWrapper>
+                ) : null}
```

Comment:
i think we should disable this feature until Minerva is implemented

Reply:
Agreed. I think we're also contemplating implementing sequential BRAVO, so we might also have that as an option too. Pending discussions.